package.json

```diff
@@ -16,12 +16,12 @@
   },
   "dependencies": {
     "@babel/compat-data": "^7.11.0",
-    "@bcrs-shared-components/bread-crumb": "^1.0.2",
+    "@bcrs-shared-components/bread-crumb": "1.0.2",
     "@bcrs-shared-components/confirm-dialog": "1.1.0",
     "@bcrs-shared-components/corp-type-module": "1.0.6",
     "@bcrs-shared-components/court-order-poa": "1.0.14",
     "@bcrs-shared-components/enums": "1.0.14",
-    "@bcrs-shared-components/interfaces": "1.0.28",
+    "@bcrs-shared-components/interfaces": "1.0.37",
```

Comment:
To import this: https://github.com/bcgov/bcrs-shared-components/blob/main/src/interfaces/date-interfaces.ts